tests/ui/wrong_self_convention.stderr

```diff
@@ -90,7 +90,7 @@ error: methods called `is_*` usually take self by reference or no self; consider
 LL |         fn is_i32(self) {}
    |                   ^^^^
 
-error: methods called `to_*` usually take self by reference; consider choosing a less ambiguous name
+error: methods called like this: (`to_*` and `~_mut`) usually take self by reference; consider choosing a less ambiguous name
```

Comment:
Should the `~` mean, that there is no `_mut` at the end? In that case, why mention it in the first place?

Reply:
Good point, agree it's better to not mention it at all. Fixed that in another commit.